### PR TITLE
Legend, new logo, chart and font tweaks, plus a few fixes

### PR DESCRIPTION
**Pull request by paulkva**
Also incorporates code changes from #81

### CODE REVIEW

File: src/main/primary_entry/css/page/map/MapView.css [hunk 1]
```diff
@@ -63,7 +63,7 @@ Markers
 .cluster-marker.permit { background-color: rgba(0, 64, 255, 0.7); }
 .cluster-marker.closed-temp { background-color: rgba(160, 160, 160, 0.7); }
 .cluster-marker.closed-perm { background-color: rgba(0, 0, 0, 0.7); }
-.cluster-marker.user { background-color: rgba(0, 192, 0, 0.7); }
+.cluster-marker.user { background-color: rgba(0, 224, 0, 0.7); color: black; }
```
Comment (paulkva) [inline src/main/primary_entry/css/page/map/MapView.css:66]: Made cluster/tooltip text for custom markers black (and added a white pin to the marker itself) to accommodate red-green colorblindness

File: src/main/primary_entry/script/primary_entry.js [hunk 2]
```diff
@@ -31,8 +35,10 @@ import "../images/red_dot.svg";
 import "../images/red_dot_t.svg";
 import "../images/red_dot_limited.svg";
 import "../images/orange_triangle.svg";
+import "../images/custom_pin.svg";
 
-
+import "../images/red_dot_expansion_permit.svg";
```
Comment (paulkva) [inline src/main/primary_entry/script/primary_entry.js:40]: We'll probably want to update these at some point to the pie-chart-like designs we discussed recently...

File: image_sources/sticker-9x1.png
Comment (Dani21) [inline image_sources/sticker-9x1.png:1]: Isn't needed for now, but eventually we'll probably want to use the updated icons

File: src/main/primary_entry/css/main.css
Comment (Dani21) [inline src/main/primary_entry/css/main.css:1]: Do we still need the Encode Sans and M Plus Rounded fonts?

Comment (paulkva) [reply src/main/primary_entry/css/main.css:1]: We don't, good catch.

File: src/main/primary_entry/html/index.html [hunk 3]
```diff
@@ -32,7 +32,8 @@
     </script>
 </head>
 
-<body>
+<!-- Hide EVERYTHING initially to allow styles to take effect - NavBar code unhides when ready -->
```
[the remark this answers is not in the document]
Comment (paulkva) [reply src/main/primary_entry/html/index.html:35]: If I remember correctly, I got rid of MiniCssExtract as part of a larger webpack change to break up the packages into smaller chunks. We could consider re-adding it, though it feels like I haven't looked at our webpack config in a while 😃

File: src/main/primary_entry/script/page/map/MapView.js [hunk 4]
```diff
@@ -32,6 +32,14 @@ export default class MapView {
         $(document).on('click', '.marker-toggle-trigger', $.proxy(this.handleMarkerRemove, this));
         $(document).on('click', '.marker-toggle-all-trigger', $.proxy(this.handleMarkerRemoveAll, this));
 
+        // this works around stacking context issues with Leaflet controls (zoom, layers, search)
+        // which would otherwise appear on top of filter dropdowns
+        $(document).on('show.bs.select', $.proxy(this.hideLeafletControls, this));
+        $(document).on('hide.bs.select', $.proxy(this.showLeafletControls, this));
```
[the remark this answers is not in the document]
Comment (paulkva) [reply src/main/primary_entry/script/page/map/MapView.js:38]: I can try it but most likely not, thanks to stacking contexts.

File: src/main/primary_entry/html/index.html (hunk 3 above)
[the remark this answers is not in the document]
Comment (Dani21) [reply src/main/primary_entry/html/index.html:35]: I'll leave the decision up to you. I'd rather not deal with webpack's config if I can 😅 Either way, the webpage loads smoothly for me now

**Review by Dani21 — APPROVED** [review]

File: src/main/primary_entry/script/page/map/MapView.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (paulkva) [reply src/main/primary_entry/script/page/map/MapView.js:38]: Actually, re-reading my own comment, I did try it 😆

**Pull request closed without merging**